app/reducers/fetchHistory.js

```diff
@@ -33,6 +33,7 @@ export default function fetchHistory(state: State = initialState, action: any) {
       };
     }
     case FetchHistory.CLEAR_HISTORY:
+    case User.LOGIN.SUCCESS:
```

Comment:
This will clear the `FetchHistory` on login, and it will force actions to run again after login.